example/01_point_example.cpp

```diff
@@ -107,6 +107,33 @@ int main()
 
 
     // Some ways of getting point values
+    typedef model::point<double, 3, cs::cartesian> point_t;
+    typedef model::ring<point_t> ring_t;
+    typedef model::PolyhedralSurface<ring_t> poly_t;
```

Comment:
This does not belong in `01_point_example.cpp`. Can you please create a completely new example?
I would prefer `08_polyhedralsurface_example.cpp` with this code?